SECURITY.md

```diff
@@ -329,7 +329,10 @@ If you aren't using gRPC on Android devices, you are most likely using `grpc-net
 
 If you are developing for Android and have a dependency on `grpc-netty`, you should remove it as `grpc-netty` is unsupported on Android. Use `grpc-okhttp` instead.
 
-If you are on a 32-bit operating system, or not on a [Transport Security supported platform](#transport-security-tls), you should use Jetty ALPN (and beware of potential issues), or you'll need to build your own 32-bit version of `netty-tcnative`.
+If you are on a 32-bit operating system, using Java 11+ may be the easiest
+solution, as ALPN was added to Java in Java 9. If on 32-bit Windows, Conscrypt
+may be an option. Otherwise you need to build your own 32-bit version of
+`netty-tcnative`.
```

Comment:
`netty-tcnative` -> [`netty-tcnative`](https://github.com/netty/netty-tcnative)

Reply:
We reference netty-tcnative _lots_ of times in this doc. It is a bad idea to make each one a link. We have a link to it in the TLS on non-Android section. I would be okay with linking to a "compiling netty-tcnative" section, but we don't have such a section and really the users are on their own at that point.

Reply:
How about then linking to https://netty.io/wiki/forked-tomcat-native.html#wiki-h2-6 ? Assuming that's the correct page for compiling netty-tcnative.

Reply:
Good idea. I had forgotten that documentation existed.